db/model/subject.js

```diff
@@ -440,9 +446,7 @@ module.exports = function subject(seq, dataTypes) {
 
               return resolve(inst);
             })
-            .catch((err) => {
-              return reject(err);
-            });
+            .catch(reject);
```

Comment:
A promise's `.catch(...)` function takes a function as an argument. The function can be an anonymous inline function like 

```.catch((err) => { reject(err); })```

or it can be a named function like

```.catch(reject)```